Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -244,15 +241,29 @@ YUI.add('ez-navigationhubview', function (Y) {
             var container = this.get('container');
 
             container.setHTML(this.template({
-                user: this.get('user').toJSON(),
                 zones: this._buildZones(),
             }));
             this._renderNavigationItems();
             this._uiSetActiveNavigation();
+            this._renderUserProfile();
             return this;
         },
 
         /**
+         * Renders user profile to the view
+         *
+         * @method _renderUserProfile
+         * @protected
+         */
+        _renderUserProfile: function () {
+            var profileView = this.get('userProfileView');
+
+            profileView.addTarget(this);
```

Comment:
this should be done as soon as soon the user profile view is instantiated. Basically, the `userProfileView` attribute should be defined with something like:

```js
            userProfileView: {
                valueFn: function () {
                    return new Y.eZ.UserProfileView({
                        bubbleTargets: this,
                    });
                }
            },
```